### PR TITLE
ci(clang-tidy-pr): remove integration tests

### PR DESCRIPTION
**Pull request by alevenberg**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/12794)

### CODE REVIEW

File: ci/cloudbuild/builds/clang-tidy.sh [hunk 1]
```diff
@@ -45,9 +45,7 @@ io::run cmake "${cmake_args[@]}" \
 io::run cmake --build cmake-out
 
 mapfile -t ctest_args < <(ctest::common_args)
-io::run env -C cmake-out ctest "${ctest_args[@]}" -LE integration-test
-
-integration::ctest_with_emulators "cmake-out"
+io::run env -C cmake-out ctest "${ctest_args[@]}"
```
Comment (coryan) [inline ci/cloudbuild/builds/clang-tidy.sh:48]: `-LE` **excludes** any test with a matching label (in this case `integration-test`) is the label. You need to revert the change to this line.

**Review by coryan — CHANGES_REQUESTED** [review]

File: ci/cloudbuild/builds/clang-tidy.sh (hunk 1 above)
Comment (alevenberg) [reply ci/cloudbuild/builds/clang-tidy.sh]: Done

**Review by coryan — APPROVED** [review]
LGTM, maybe wait until @devbww and @dbolduc have a chance to chime in.

**Review by dbolduc — APPROVED** [review]

**Review by devbww — APPROVED** [review]
